### PR TITLE
Intro Python PR

### PR DESCRIPTION
**Pull request by MFalahee**

### CODE REVIEW

File: src/02_datatypes.py [hunk 1]
```diff
@@ -15,7 +15,10 @@
 
 # YOUR CODE HERE
 
+print(int(y) + x); 
 
 # Write a print statement that combines x + y into the string value 57
 
+print(f'{x}{y}')
```
Comment (andrewlowell) [inline src/02_datatypes.py:22]: I bet the intention was `print(y + str(x))`, but no biggie :)

File: src/03_modules.py [hunk 2]
```diff
@@ -11,21 +11,33 @@
...
+
 # Print the current working directory (cwd):
 # YOUR CODE HERE
 
+print('Current Working Directory', os.getcwd())
+
 # Print out your machine's login name
 # YOUR CODE HERE
+
+print("Login Name", os.getlogin())
```
Comment (andrewlowell) [inline src/03_modules.py:43]: I appreciate that every print has a description.

File: src/08_comprehensions.py [hunk 3]
```diff
@@ -12,13 +12,19 @@
...
+    y.append(i)
+
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
 
+for i in range(10):
```
Comment (andrewlowell) [inline src/08_comprehensions.py:25]: These work, of course, but the problem is list comprehensions and these are...not list comprehensions :)

They're looking for `y = [i * i * i for i in range(10)]`

Comment (MFalahee) [reply src/08_comprehensions.py:25]: So the distinction is that a list comprehension always occurs within the list reference?

Comment (andrewlowell) [reply src/08_comprehensions.py:25]: As far as I can tell, you're right and it's syntactic sugar for list creation, no extra functionality, so I definitely won't make you rewrite them On the off chance you ever get asked about them, you'll know :)

File: src/02_datatypes.py (hunk 1 above)
Comment (andrewlowell) [reply src/02_datatypes.py:22]: Err, and of course I mean `print(str(x) + y)` :sweat_smile:

**Review by andrewlowell — APPROVED** [review]
Great job!

**Pull request merged**